src/h5web/vis-packs/core/scales.ts

```diff
@@ -3,47 +3,59 @@ import {
   scaleLog,
   scaleSymlog,
   scaleSqrt,
-  PickScaleConfig,
+  PickScaleConfigWithoutType,
 } from '@visx/scale';
 import type { IconType } from 'react-icons/lib';
-import type { AxisScale } from './models';
+import type { AxisScale, GammaScaleConfig } from './models';
 import { ScaleType } from './models';
-import { MdSort, MdFilterList } from 'react-icons/md';
+import { MdSort, MdFilterList, MdFlare } from 'react-icons/md';
 import SqrtIcon from '../../toolbar/controls/ScaleSelector/SqrtIcon';
 import MdGraphicEqRotated from '../../toolbar/controls/ScaleSelector/MdGraphicEqRotated';
+import { gammaScale } from './gammaScale';
 
 interface H5WebScale {
-  createScale: (
-    config: Omit<PickScaleConfig<ScaleType, number>, 'type'>
-  ) => AxisScale;
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  createScale: (config: any) => AxisScale;
```

Comment:
Some improvement to be made here. I could not find a working solution.

I tried using `createScale` from `@visx` as well but I ran into other TS problems... (overrides were not working as intended and using a template resulted in a infinite TS recursion :shrug: )